tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -1119,8 +1121,9 @@ def compile_wasm_to_aot(wasm_tempfile, aot_tempfile, runner, opts, r, output = '
         cmd.append("--enable-gc")
         cmd.append("--enable-tail-call")
 
-    if opts.memory64:
-        cmd.append("--enable-memory64")
+    # FIXME: uncomment me when wamrc supports --enable-memory64 switcher
+    # if opts.memory64:
+    #     cmd.append("--enable-memory64")
```

Comment:
Can remove these lines or keep unchanged, since `--enable-memory64` is also not added for wamrc in AOT memory64 PR and these lines will be removed:
https://github.com/bytecodealliance/wasm-micro-runtime/pull/3362/files#diff-61f1e858cf7f152d552ec7d555ca279fd9435f80bf6d953631ce4996c6573ccc